test/test.ts

```diff
@@ -28,8 +33,48 @@ describe("Test Excel Custom Functions", function () {
         const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js`;
         const devServerPort = parseNumber(process.env.npm_package_config_dev_server_port || 3000);
             await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, 
-                devServerCmd, devServerPort, undefined, undefined, undefined, false /* enableDebugging */);
-    }),
+                devServerCmd, devServerPort, undefined, undefined, undefined, true /* enableDebugging */);
+    });
+    describe("Test Debugger", function () {
+        let ws;
+        before("Open websocket connection to Debugger", async function () {
+            this.timeout(12000);
+            await sleep(10000);
+            const url = 'ws://localhost:9229/runtime1';
+            ws = new WebSocket(url);
+            await sleep(1000);
+
+            ws.on('error', (err) => {
+                assert.fail(`Connection could not be established. ${err}`);
+            });
+    
+            ws.on('message', (data) => {
+                assert.equal(JSON.parse(data).error, undefined);
+            });
+        }),
+        it("enable debugging", function () {
+            ws.send(JSON.stringify({
```

Comment:
Think about using helper functions. You can have a function to send a CDP message (JSON string). This could be used by functions which perform and action by sending one or more CDP messages. You can have a function to compose a CDP message by taking the method string and any parameters, composing the JSON object and returning the JSON string.